### ISSUE TITLE
No completions returned on Windows using last night's build or latest flutter

### ISSUE THREAD

**Issue by DanTup** [dart-lang/sdk#32186]
I was trying to test something with the nightly build of the SDK (from http://gsdview.appspot.com/dart-archive/channels/be/raw/latest/sdk/) earlier and I was getting no completions back on Windows. Tried same thing on my Mac and it was fine.

Just now, noticed the same thing with latest flutter Code.

Log of me trying to complete on `print`:

[analyzer.txt](https://github.com/dart-lang/sdk/files/1728744/analyzer.txt)

@bwilkerson

**Comment by devoncarew** [comment]
It would be awesome to bisect this; I understand you can't build on windows just yet.

**Comment by devoncarew** [comment]
also /cc @scheglov

**Comment by DanTup** [comment]
Actually, the problem I had running from source on Windows a few days ago was the same as this - I was getting no results back. I figured I hadn't set things up properly (I had errors about my VS install during `gclient sync`). Now it's looking like it was probably the same issue.

**Comment by devoncarew** [comment]
It's not clear to me how the analysis server's integration tests for code completion could pass on the windows bots w/ this issue.

**Comment by DanTup** [comment]
I thought that too; so I'm hoping this is just something really wonky at my end, but since I've seen it with last nights and latest flutter but as soon as I switch to another it works, I'm not so sure. If someone on Windows can pull latest Flutter and test it there, that should help.

**Comment by DanTup** [comment]
@devoncarew @bwilkerson

Oh man, I just tested this from source with/without my Windows drive letter change and it looks bad. Completions without it, no results with it. I don't know why this isn't breaking tests (or why it fixes the analysis issues yet breaks completion).

I've got to run shortly, someone can revert my change and I can dig into it next week.

https://github.com/dart-lang/sdk/pull/32133
